src/components/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import bgBannerDesktop from '@/assets/image-hero.jpg'
+import bgBannerMobile from '@/assets/image-header.jpg'
+import bgFeaturedTwoDesktop from '@/assets/zx7-desktop.jpg'
+import bgFeaturedTwoMobile from '@/assets/zx7-mobile.jpg'
+import BannerSection from './subcomponents/BannerSection'
+import CategoriesSection from './subcomponents/CategoriesSection'
+import FeaturedSection from './subcomponents/FeaturedSection'
+
+const Homepage = () => {
+  const dynamicBackgroundImagesStyle = () => {
```

Comment:
is it possible not to use inline-style?

Reply:
I've explored other ways to do this but this is the solution I arrived at where I did not need to install additional libraries such as styled-components to integrate JS imports to CSS.